guides/developer/extending/integrate-your-CI/index.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: single
+title:  "Integrate your CI"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+At Netflix, we built CI integrations to enable features like CI in Spinnaker, fetch artifact metadata and more. You can see the full list of CI features [here](/guides/user/managed-delivery/CI-features).
+
+This guide will explain how to add an integration to your own CI provider, so you'll be able to enjoy the new features we provide.
```

Comment:
```suggestion
This guide will explain how to add an integration to other CI providers, so that users leveraging those providers can enjoy the new features we provide.
```